bin/postinstall.js

```diff
@@ -235,7 +235,7 @@ new Promise((resolve, reject) => {
 .then(() => {
   // Copy and configure the stub executable.
 
-  switch(process.platform) {
+  switch (process.platform) {
```

Comment:
It looks like we can enforce this space with the `keyword-spacing: ["error"]` eslint rule.